shell/list/node.vue

```diff
@@ -135,6 +137,28 @@ export default {
         this.$forceUpdate();
       }
     },
+    displayTaintsAndLabels(row) {
```

Comment:
I'm not a fan of using method calls in the template - I would have added a new computed for the table rows which returned a new rows array with each row extended to have the computed properties for displayLabels and the this boolean.

You know then that the computed is only re-evaluated when `rows` changes.

Reply:
Agree with Neil's take on the `tr` approach.

Just create a `parsedRows` computed and handle all of your data there. Something like:

```
this.rows.forEach((row)=>{
  const out = []
  for (const k in row.labels) {
    const [prefix] = k.split('/');
    if (!SYSTEM_LABELS.includes(prefix)) {
      out.push(`${ k }=${ row.labels[k] }`);
    }
  }
  rows.parsedLabels = out;
  
  row.displayTaintsAndLabels = ....
})
```

Then you just replace the `rows` variable on the `ResourceList` with `parsedRows` and adapt the rest accordingly.